openml-lightgbm/src/test/java/com/feedzai/openml/provider/lightgbm/TestSchemas.java

```diff
@@ -107,7 +107,7 @@
      * "uuid","cat1_generator","cat2_generator","cat3_generator","num1_float","num2_double","num3_int",
      * "cat1_string","cat2_string","cat3_string"
      */
-    static final DatasetSchema CATEGORICALS_SCHEMA_LABEL_AT_END = new DatasetSchema(
+    public static final DatasetSchema CATEGORICALS_SCHEMA_LABEL_AT_END = new DatasetSchema(
```

Comment:
Same here: what's the need for the public visibility here?

Reply:
I think for the AutoML tests we exposed a schema with numericals and another with numericals + categoricals.

Reply:
This restores that.